dataquality/schemas/semantic_segmentation.py

```diff
@@ -115,25 +114,27 @@ def contours_opencv(self) -> List[np.ndarray]:
             contours.append(np.array(pixels))
         return contours
 
-    @property
-    def contours_json(self) -> List:
+    def contours_json(self, image_id: int) -> Dict[int, List]:
         """Deserialize the contours as a JSON
 
+        In the backend we store polygon contours per image, so we need
+        to keep a reference to which image the polygon belongs to.
+
         Example:
             polygon = Polygon(
                 contours=[
                     Contour(pixels=[Pixel(x=0, y=0), Pixel(x=0, y=1)]),
                     Contour(pixels=[Pixel(x=12, y=9), Pixel(x=11, y=11)])
                 ]
             )
-            polygon.contours_opencv
-            >>> [[[0, 0], [0, 1]], [[12, 9], [11, 11]]]
+            polygon.contours_json("123")
```

Comment:
Arg is an str? Should be `int`, right?
```suggestion
            polygon.contours_json(123)
```